circle.yml

```diff
@@ -0,0 +1,7 @@
+machine:
+  node:
+    version: 6
+test:
+  override:
+    - npm run lint
+    - npm run coverage
```

Comment:
you can add a threshold on coverage. for example set the minimum to 80% and fail if the result is under that limit